### DISCUSSION TITLE
Swipe Slide Link

### DISCUSSION THREAD

**Discussion by BGillham** [nolimits4web/swiper#4911] [Q&A]
I have entered a Link for each Slide in the Swiper Slider and for some reason they do not show... Perhaps I am just doing something wrong but how do I get the Links to work in each Slide? Sorry if this is a stupid question but I am new to 2SXC Components.
I have also found out that Title / Description / Links are only showing when the "Paralax" mode is selected!
The links open in a new tab regardless of the option selected...
![image](https://user-images.githubusercontent.com/19285313/131853231-50cb99d0-ca0b-486f-917d-4d1445b77090.png)

**Comment by vltansky** [comment]
This question should be addressed to 2SXC Components not to Swiper.

**Reply by BGillham** [reply]
Thank you but I thought that this forum is monitored and run by 2SXC and the question is about the Component. There are many Users of these Modules in this Forum so I assume that somebody here could answer this question.
